R/pkg/R/generics.R

```diff
@@ -396,67 +397,81 @@ setGeneric("agg", function (x, ...) { standardGeneric("agg") })
 #' @param object x a SparkDataFrame or a Column
 #' @param data new name to use
 #' @return a SparkDataFrame or a Column
+#' @noRd
 NULL
 
 #' @rdname arrange
 #' @export
+#' @noRd
 setGeneric("arrange", function(x, col, ...) { standardGeneric("arrange") })
 
 #' @rdname as.data.frame
 #' @export
+#' @noRd
 setGeneric("as.data.frame",
            function(x, row.names = NULL, optional = FALSE, ...) {
              standardGeneric("as.data.frame")
            })
 
 #' @rdname attach
 #' @export
+#' @noRd
 setGeneric("attach")
 
 #' @rdname cache
 #' @export
+#' @noRd
 setGeneric("cache", function(x) { standardGeneric("cache") })
 
 #' @rdname checkpoint
 #' @export
+#' @noRd
 setGeneric("checkpoint", function(x, eager = TRUE) { standardGeneric("checkpoint") })
 
 #' @rdname coalesce
 #' @param x a Column or a SparkDataFrame.
```

Comment:
waiting - having these here is intentional? if a function supports different type we don't want the documentation to go to DataFrame.R because then Column.R will look empty.